pdns/pdnsutil.cc

```diff
@@ -367,7 +376,7 @@ static int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, con
       stringtok(parts, rr.content);
 
       if(parts.size() < 7) {
-        cout<<"[Warning] SOA autocomplete is deprecated, missing field(s) in SOA content: "<<rr.qname<<" IN " <<rr.qtype.toString()<< " '" << rr.content<<"'"<<endl;
+        cout << "[Info] SOA autocomplete is deprecated, missing field(s) in SOA content: " << rr.qname << " IN " << rr.qtype.toString() << " '" << rr.content << "'" << endl;
```

Comment:
Shouldn't this be an error, now that we really don't autocomplete any more?

Reply:
The output is:
```
[Error] SOA lookup failed: Out of range exception parsing 'cryptokeys.ds9a.nl ahu.ds9a.nl 2009071301 14400 3600 604800'
[Info] SOA autocomplete is deprecated, missing field(s) in SOA content: cryptokeys.org IN SOA 'cryptokeys.ds9a.nl ahu.ds9a.nl 2009071301 14400 3600 604800'
```
This is now additional information

Reply:
ah, cool :)